<launch>
  <node name="usb_camera_node" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <!-- <param name="image_width" value="1280" />
    <param name="image_height" value="720" /> -->
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <remap from="/usb_cam/image_raw" to="/camera/image_raw"/>
  </node>

  <node name="image_view_node" pkg="image_view" type="image_view" respawn="false" output="screen" >
    <remap from="image" to="/usb_camera_node/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <node name="ros_ball_tracker" pkg="turtlesim_cleaner" type="ros_ball_tracker" output="screen" >
        <remap from="/usb_cam/image_raw" to="/usb_camera_node/image_raw"/>
  </node>

  <!-- <node name="image_view2_node" pkg="image_view" type="image_view" output="screen" >
        <remap from="image" to="/image_converter/output_video"/>
  </node> -->

</launch>